lib/DocApi.php

```diff
@@ -403,7 +411,7 @@ public function createDoc($doc)
      *
      * @throws \DocRaptor\ApiException on non-2xx response
      * @throws \InvalidArgumentException
-     * @return array of \SplFileObject, HTTP status code, HTTP response headers (array of strings)
+     * @return array of string, HTTP status code, HTTP response headers (array of strings)
```

Comment:
This change is a BC break and my code broke because of it. Please do not change the public API in patch versions. Thanks!